### ISSUE TITLE
Compiled Version 28.05.2019

### ISSUE THREAD

**Issue by 2play** [KaosOverride/CapriceRPI#9]
hi @KaosOverride 
I just compile from source on retropie and still gives

 ./capriceRPI -v
======================
CapriceRPI Start:
======================
 V1.3c (CaPriCe32 v4.1.0) Release
 KaosOverride - 2017
 Caprice for RPI!
...

should i manually compile to get the 1.3d or its just a typo in the version info?

**Comment by KaosOverride** [comment]
Yes, is just a typo, I forget to update the version string. If --help gives
the "parados" options then is the new version, there are no other changes.
Will try to correct it. I'm sorry

**Comment by 2play** [comment]
no worries @KaosOverride 
thank you for your kind reply!
when its done please let me know so I recompile.
Please also put a push request to retropie setup as they are little riggid with changes

**Comment by 2play** [comment]
@KaosOverride The compile info points to makefile.unix
Should I try the dynamic or the .old?

**Comment by KaosOverride** [comment]
Sorry, the README.TXT is just the original Caprice32 4.1.0 and is there for
historical purpouses. Just the main Makefile is enought for raspberry
Raspbian compilation.

That Makefile just has an issue with libpng12, but is safe to remove the
dependencie because the libpng was just used for screen snapshoot to png
file, which was removed from the times this emulator was for the GP2X
handheld. This will be updated when possible.

**Comment by 2play** [comment]
@KaosOverride Tx i removed the libpng12 and compiled
Only a warning at the end
`cap32.cpp:(.text+0x2330): warning: the use of `mktemp' is dangerous, better use `mkstemp' or `mkdtemp'`

Im testing on a pi4 with fksm enabled. 
Edited:
I love how capriceRPI works. The menu the autoboot. I do hope you can add the + support wiht the gx4000

It will be also nice to have a SAVE OPTIONS or SAVE CONFIG in the MENU.

In the menu its missing the 464D which as you said can be done by command option.

My last comments are that I noticed that your binary boots fine at full screen even on pi4 which is great.
BUT if i use the model and mem option its shrunk on the top left quarter of the screen IF i create an entry as option 2 below
any idea?

example
```
1. **capricerpi** = "/opt/retropie/emulators/capricerpi/capriceRPI  %ROM%"
2. **capricerpi-6128** = "/opt/retropie/emulators/capricerpi/capriceRPI --model 6128 --mem 576 --video 3 %ROM%"
```

Sometimes option 2 wont start at all if i change before rom boot.
I think it just weird and prob to the pi4 setup.

I live with using 
**capricerpi** = "/opt/retropie/emulators/capricerpi/capriceRPI --model 6128 --mem 576 --video 3 %ROM%"
:-)
tx again for all your work.
Looking forward to any updated code/+/gx4000 options

**Comment by KaosOverride** [comment]
I'm sorry I don't have yet a RPI4, maybe is some issue with the SDL libs
(Not the SDL dev libs, but the runtime)

Check if there is any difference executing this lines before calling the
emulator

export SDL1_VIDEODRIVER="dispmanx"

or

export SDL1_VIDEODRIVER="fbcon"

**Comment by 2play** [comment]
thank you @KaosOverride 
there are 2 ways to test without rpi4
tyou can enable fkms on rpie with the  raspi-config
or if you are on use a pi4 retropie clean image, boot on pi3 board and just from rpi setup you can remove and reinstall the core packs so you can have the pi3 fkms packs. 

just run these commands to set env sdl parameter?
from cli
is this permanent or can be reversed after being set?

You auto boot and video full screen borders or not working fine. Just when i create a new entry in emulators.cfg it gets messed but i can use the main as 6128 with 576 mem . All good and very happy. Most roms are in dks format anyway.
thank you for taking the time and replying to me as always
